clippy_dev/src/main.rs

```diff
@@ -87,9 +87,31 @@ fn print_lints() {
 
 fn update_lints(update_mode: &UpdateMode) {
     let lint_list: Vec<Lint> = gather_all().collect();
+
     let usable_lints: Vec<Lint> = Lint::usable_lints(lint_list.clone().into_iter()).collect();
     let lint_count = usable_lints.len();
 
+    let mut sorted_usable_lints = usable_lints.clone();
+    sorted_usable_lints.sort_by_key(|lint| lint.name.clone());
+
+    std::fs::write(
+        "../src/lintlist/mod.rs",
```

Comment:
After the fact review:

Shouldn't this also be done with the `replace_region_in_file` function and then only update the content in the `ALL_LINTS` array? When a PR adds a lint, this file needs to get updated like `README.md`, ... If the update is done with the `replace_region_in_file` function (like below), travis will fail if it's not updated. cc @phansch